android/build.gradle

```diff
@@ -35,14 +35,14 @@ android {
 
 dependencies {
     implementation 'com.facebook.react:react-native:+' // From node_modules
-    implementation 'com.github.zoom-us-community:jitpack-zoom-us:5.13.1.11014'
+    implementation 'com.github.zoom-us-community:jitpack-zoom-us:5.13.10.12578' // 5.13.10.12577
```

Comment:
Using `5.13.10.12578` instead of `5.13.10.12577` because of wrong build at `5.13.10.12578` on JitPack (once JitPack builds from tag it cannot be overriden).